package.json

```diff
@@ -2,11 +2,14 @@
   "private": true,
   "scripts": {
     "build": "vitruvius build --src src --dest lib --ignore-packages \"**/integration-tests/**\"",
-    "clean": "rimraf ./packages/*/lib",
+    "prebuild": "rimraf ./packages/*/lib",
     "lint": "eslint . --cache",
-    "postinstall": "yarn run clean --silent && yarn run build --silent",
+    "postinstall": "yarn build --silent",
     "precommit": "lint-staged",
-    "test": "jest",
+    "setup": "lerna clean --yes && lerna bootstrap",
```

Comment:
I couldn't get storybook to run without it because of some missing dependencies. Hence I added `integration/*` to the managed packages and use `lerna bootstrap` to link them up/hoist the dependencies into it. I am surprised it worked before - which steps would I have followed to make it work? I have `0.27.5` now, but can't guarantee I had it when I started working on this PR.